src/com/javacore/lesson06/Input.java

```diff
@@ -0,0 +1,117 @@
+package com.javacore.lesson06;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+public class Input {
+
+    ArrayList arrayList;
+
+    private BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+
+    public Input(ArrayList arrayList) {
+        this.arrayList = arrayList;
+    }
+
+    public boolean menu() throws IOException {
+
+        while (true) {
+            System.out.println("1 - Menu");
+            System.out.println("2 - Exit");
+            System.out.print("Enter number: ");
+            switch (Integer.parseInt(bufferedReader.readLine())) {
+                case 1 -> this.enter();
+                case 2 -> {
+                    return true;
+                }
+                default -> {
+                    System.out.println("Try again ");
+                    continue;
+                }
+            }
+        }
+    }
+
+    public void enter() throws IOException {
+
+        int num;
+
+        System.out.print("Enter Order ID: ");
+        num = Integer.parseInt(bufferedReader.readLine());
+        Order order = new Order(num);
+
+        if (arrayList.contains(order)) {
+            Order order1 = (Order) arrayList.get(arrayList.indexOf(order));
```

Comment:
better to avoid naming with numbered prefixes/postfixes
just think what is it and name it by words.
existingOrder, foundOrder etc